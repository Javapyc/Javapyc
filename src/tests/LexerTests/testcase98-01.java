2 + 3 /* Comment */ ; true || false

/*  Output - Invalid Program
	Interger, 2
	Operator, +
	Interger, 3
	Delimiter, ;
	ReservedWord, true
	Operator, ||
	ReservedWord, false
*/