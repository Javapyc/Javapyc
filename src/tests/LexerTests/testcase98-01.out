Integer, 2
Operator, +
Integer, 3
Delimiter, ;
ReservedWord, true
Operator, ||
ReservedWord, false
